config/config_dev.yaml

```diff
@@ -0,0 +1,71 @@
+log_directory: /report_service/logs/final
+#log_directory: /home/kaipak/datasets/report_service/logs/final
+profile_name: saml-pub
+s3_bucket: pds-web-analytics-cloud
+s3_logdir: logs_dev
+#s3_logdir: logs_dev
```

Comment:
In this case, the commented-out line is identical to the line above it, so definitely delete this line